sql/parser/eval.go

```diff
@@ -1033,10 +1033,8 @@ func evalCastExpr(expr *CastExpr) (Datum, error) {
 		return s, nil
 
 		// TODO(pmattis): unimplemented.
-		// case *BitType:
 		// case *DecimalType:
 		// case *DateType:
-		// case *TimeType:
```

Comment:
Why not? My thought was that most of the logic for date and time will be the same.